### ISSUE TITLE
[Bug]: Crash game because of an attempt to grow a plant with bone meal

### ISSUE THREAD

**Issue by BlackAures1** [CorgiTaco/BYG#602]
### Mod Version

byg-1.3.5.jar

### Mod Loader

Forge

### Mod Loader Version

36.2.22

### What happened?

HI! Some plant broke itself when I try to bonemeal it but still reported the bone meal as success therefor it crashed since the block was null.
![image](https://user-images.githubusercontent.com/43911230/146608831-ce8692ed-fff2-4b15-9b15-d4c41e246505.png)
[crash-2021-12-17_23.59.38-server.txt](https://github.com/AOCAWOL/BYG/files/7737783/crash-2021-12-17_23.59.38-server.txt)

### latest.log

[latest.log](https://github.com/AOCAWOL/BYG/files/7737776/latest.log)

**Comment by TowardtheStars** [comment]
I encounted the same bug. And I did a little analysis.

According to log, the NPE takes place here:
![屏幕截图 2022-01-26 023650](https://user-images.githubusercontent.com/29396015/151038305-1b39e127-47aa-41a2-8444-54528e055870.png)

`Chunk::heightmaps` is an instance of `Map`, so the problem might be on `Map::get`, which can return a null value. So I guess the key passed to it doesn't exist.

In log line 46:
`at corgiaoc.byg.common.properties.blocks.BYGGrassBlock.func_225535_a_(BYGGrassBlock.java:55) ~[?:1.3.5] {re:classloading}`
This is the `performBonemeal` method for class `BYGGrassBlock`. According to the log, the problem is the [line 55](https://github.com/AOCAWOL/BYG/blob/5e45f6afb8567b609c91c613bda97533bfe1f980/src/main/java/corgiaoc/byg/common/properties/blocks/BYGGrassBlock.java#L55).
In the line, the mod uses `Features.PATCH_GRASS_NORMAL.place(...)` to generate grass and flowers. 
However, the method called here tries to query a non-existing height map of `HeightMap.Type`: `WORLD_SURFACE_WG` in generated chunks. This might be the cause of the NPE.

Maybe the height map of the type `WORLD_SURFACE_WG` is deleted from chunk files after worldgen, I'm not sure about that. But I can't find any of its presence in any mca files:

![The original point of a world](https://user-images.githubusercontent.com/29396015/151037474-5e926d4e-917b-4ba8-88a9-5f25d148263f.png)

The original point of a world, no `WORLD_SURFACE_WG` key here. It might be a pregen-only feature and will never appear again once a chunk is generated.
